include/capo/utils/file_utils.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <fstream>
+#include <string>
+#include <vector>
+
+namespace capo::utils {
+
+inline std::vector<std::byte> fileBytes(std::string const& path) {
```

Comment:
Where was this before?

Reply:
Inside pcm.cpp; I think I had to pull it to make some internal changes but it's no longer required (even then it's not a bad idea to have it in a separate file)